app/views/editor/application/_navigation.html.haml

```diff
@@ -0,0 +1,14 @@
+.page-header
+  %nav.navbar.navbar-default
+    .container-fluid
+      .navbar-header
+        .navbar-brand "#{current_site.name}"
+
+      %ul.nav.navbar-nav.navbar-right
+        = active_link_to 'カテゴリ', editor_categories_path, wrap_tag: 'li'
+        = active_link_to '記事',     editor_posts_path,      wrap_tag: 'li'
+        = active_link_to '固定ページ', editor_fixed_pages_url, wrap_tag: 'li'
```

Comment:
In some situations `_path` doesn't work well (ex. CORS).
But `_url` always works.
So I'll replace `_path` with `_url`